### ISSUE TITLE
url limit off by one if showCompact is true

### ISSUE THREAD

**Issue by mafri27** [Icinga/icingadb-web#580]
## Describe the bug

I've tried to add a service-view to the dashboard while using the "limit=5" option. But I get 6 results listed. The generated URL is OK,

## To Reproduce

https://server/icingaweb2/icingadb/services?service.state.soft_state!=0&limit=5

will get 5 results

https://server/icingaweb2/icingadb/services?service.state.soft_state!=0&limit=5&showCompact=1

will get 6 results

## Screenshots
![dashboard2](https://user-images.githubusercontent.com/47006694/177221673-387d2816-c086-42ce-a49d-1c69b830ccea.png)
![dashboard_setting](https://user-images.githubusercontent.com/47006694/177221674-ed3ccdda-4b38-4f7b-9648-d364cb19cdc5.png)

## Your Environment

```
$ icingacli --version
Icinga Web 2  2.11.0    
Git Commit    46998464ea9f9e3c2889d98f9fb6df639d972550 
PHP Version   7.4.28    
MODULE       VERSION   
grafana      1.4.2     
icingadb     1.0.0     
lazaroblanc  1.0.0     
monitoring   2.11.0
```

```
$ lsb_release -d
Description:	Debian GNU/Linux 11 (bullseye)
```

```
$ icinga2 --version
icinga2 - The Icinga 2 network monitoring daemon (version: r2.13.4-1)

System information:
  Platform: Debian GNU/Linux
  Platform version: 11 (bullseye)
  Kernel: Linux
  Kernel version: 5.10.0-8-amd64
  Architecture: x86_64

Build information:
  Compiler: GNU 10.2.1
  Build host: runner-hh8q3bz2-project-298-concurrent-0
  OpenSSL version: OpenSSL 1.1.1n  15 Mar 2022
```

```
$ icinga2 feature list
Disabled features: 
Enabled features: api checker ido-pgsql influxdb mainlog syslo
```

```
$ icinga2 daemon -C
[2022-07-01 15:13:29 +0000] information/cli: Icinga application loader (version: r2.13.4-1)
[2022-07-01 15:13:29 +0000] information/cli: Loading configuration file(s).
[2022-07-01 15:13:29 +0000] information/ConfigItem: Committing config item(s).
[2022-07-01 15:13:29 +0000] warning/Zone: The Zone object 'checker' has more than two endpoints. Due to a known issue this type of configuration is strongly discouraged and may cause Icinga to use excessive amounts of CPU time.
[2022-07-01 15:13:29 +0000] information/ApiListener: My API identity: icinga-master-1.hetzner.com
[2022-07-01 15:13:39 +0000] information/WorkQueue: Icinga/icingadb-web#9 (ApiListener, RelayQueue) items: 0, rate:  0/s (0/min 0/5min 0/15min);
[2022-07-01 15:13:39 +0000] information/WorkQueue: Icinga/icingadb-web#10 (ApiListener, SyncQueue) items: 0, rate:  0/s (0/min 0/5min 0/15min);
[2022-07-01 15:13:39 +0000] information/WorkQueue: Icinga/icingadb-web#5 (DaemonUtility::LoadConfigFiles) items: 32, rate: 174.933/s (10496/min 10496/5min 10496/15min);
[2022-07-01 15:13:39 +0000] information/WorkQueue: Icinga/icingadb-web#6 (InfluxdbWriter, influxdb) items: 0, rate:  0/s (0/min 0/5min 0/15min);
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 106205 Notifications.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 IcingaApplication.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 10698 Dependencies.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 15 HostGroups.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 10474 Hosts.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 FileLogger.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 118 Downtimes.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 SyslogLogger.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 28 Comments.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 CheckerComponent.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 InfluxdbWriter.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 3 Zones.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 2 IdoPgsqlConnections.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 6 Endpoints.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 2 ApiUsers.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 ApiListener.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 177 CheckCommands.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 1 UserGroup.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 15 ServiceGroups.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 15 Users.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 251208 Services.
[2022-07-01 15:13:41 +0000] information/ConfigItem: Instantiated 3 NotificationCommands.
[2022-07-01 15:13:41 +0000] information/ScriptGlobal: Dumping variables to file '/var/cache/icinga2/icinga2.vars'
[2022-07-01 15:13:41 +0000] information/cli: Finished validating the configuration file(s).

**Comment by nilmerg** [comment]
Thanks for the report! Though, this is already fixed as part of #540. icinga-php-library v0.9.1 will be released soon which includes the fix.

**Comment by nilmerg** [comment]
Duplicate of #540.
